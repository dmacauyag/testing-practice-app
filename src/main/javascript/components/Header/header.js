import React from 'react'

const Header = () => {
  return (
    <header>
      <div>I am a header.</div>
    </header>
  )
}

export default Header 
