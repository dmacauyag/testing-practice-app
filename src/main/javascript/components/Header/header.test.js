import React from 'react'
import Header from './header'
import renderer from 'react-test-renderer'

describe('the Header component', () => {
  it('should correctly render a header element', () => {
    const component = renderer.create(
      <Header />
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
