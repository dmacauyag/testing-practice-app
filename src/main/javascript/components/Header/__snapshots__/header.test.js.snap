// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the Header component should correctly render a header element 1`] = `
<header>
  <div>
    I am a header.
  </div>
</header>
`;
